presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/RowExpressionFormatter.java

```diff
@@ -35,9 +38,13 @@
 public final class RowExpressionFormatter
 {
     private final ConnectorSession session;
+    private final FunctionManager functionManager;
+    private final StandardFunctionResolution standardFunctionResolution;
 
-    public RowExpressionFormatter(ConnectorSession session)
+    public RowExpressionFormatter(ConnectorSession session, FunctionManager functionManager)
```

Comment:
Let's use `StandardFunctionResolution` for the constructor; so it's more general